code/modules/mob/living/carbon/human/species/station/prometheans_vr.dm

```diff
@@ -27,6 +21,9 @@
 		/mob/living/carbon/human/proc/shapeshifter_select_hair,
 		/mob/living/carbon/human/proc/shapeshifter_select_hair_colors,
 		/mob/living/carbon/human/proc/shapeshifter_select_gender,
+		/mob/living/carbon/human/proc/shapeshifter_select_wings,
+		/mob/living/carbon/human/proc/shapeshifter_select_tail,
+		/mob/living/carbon/human/proc/shapeshifter_select_ears,
```

Comment:
Can you test these to make sure they look proper, visually, on Prometheans? I did them while doing Proteans, but haven't tested whether they assume the proper colors and stuff on Prometheans. They might not.